scripts/secnetperf-helpers.psm1

```diff
@@ -6,6 +6,13 @@
 Set-StrictMode -Version "Latest"
 $PSDefaultParameterValues["*:ErrorAction"] = "Stop"
 
+
+$psVersion = $PSVersionTable.PSVersion
+if ($psVersion.Major -lt 7) {
+    $isWindows = $true
```

Comment:
Can we just use `$IsWindows` everywher? PS isn't actually case sensitive AFAIK anyways.